test/lib/Launcher.ts

```diff
@@ -130,6 +132,7 @@ export class Launcher {
                 res.status(200).send("OK");
 
                 eventBus.publish(new DataEvent(req.body.trigger, req.body.data));
+                eventBus.publish(new DataEvent("__headers", req.headers));
```

Comment:
why not enhance the data event with the headers?

Reply:
I wanted to leave the original event as because I thought the headers would not be there but as it is the webhook they are there anyway.